pkg/apis/compliance/v1alpha1/tailoredprofile_types.go

```diff
@@ -15,6 +15,9 @@ const PruneOutdatedReferencesAnnotationKey = "compliance.openshift.io/prune-outd
 // RuleLastCheckTypeChangedAnnotationKey is the annotation key used to indicate that the rule check type has changed, store its previous check type
 const RuleLastCheckTypeChangedAnnotationKey = "compliance.openshift.io/rule-last-check-type"
 
+// ExtendedProfileGuidLabel is a label used to store the unique ID of the profile being extends
+const ExtendedProfileGuidLabel = "compliance.openshift.io/extended-profile-unique-id"
```

Comment:
For consistency this can should also be renamed to:
```suggestion
const ExtendedProfileGuidLabel = "compliance.openshift.io/extended-profile-guid"
```